wordle-web/components/LetterResult.vue

```diff
@@ -1,56 +1,62 @@
 <template>
-  <div class="letter-box" :class="color">
+  <v-card
+    :height="boxSize"
+    :width="boxSize"
+    :color="letter.color"
+    flat
+    :class="[
+      clickable ? '' : 'no-pointer',
+      'align-center d-flex justify-center',
+    ]"
+    @click="onClicked()"
```

Comment:
You can set a property here to add in drop shadow to your keyboard. Also this is where you should be adding in gradient for letters and keys. This will involve using css. [[Docs]](https://vuetifyjs.com/en/features/theme/#custom-theme-colors)